charts/prometheus-operator.yaml

```diff
@@ -148,10 +148,21 @@ grafana:
 
   grafana.ini:
     server:
-      root_url: /grafana
+      root_url: '__url__(https://{{ grains.metalk8s.control_plane_ip }}:8443/grafana)'
```

Comment:
What's this `__url__` thing doing?

Reply:
OK I looked at the code. The name is bad (we're expecting something special about URLs), `__escape__` could be more explicit.

Reply:
It's just one of our magical ways of substituting unquoted URLs from jinja to YAML.

See: https://github.com/scality/metalk8s/blob/development/2.6/charts/render.py#L144

Reply:
> OK I looked at the code. The name is bad (we're expecting something special about URLs), `__escape__` could be more explicit.

Ok, but wouldn't fix it in this PR since the Prometheus-operator charts make use of this entry

I will fix this probably next week when I am about to bump the versions for Dex and Prometheus-operator chart